### ISSUE TITLE
Retroarch, Lumia 950XL and Bluetooth

### ISSUE THREAD

**Issue by namibiz** [basharast/RetroArch-ARM#4]
Hi. I'm testing version 1.10.3 on a Lumia 950 XL and I've found and issue with Bluetooth.

I've paired a Xbox Controller and a generic joystick via OS bluetooth, but Retroarch don't recognized them, even with all combinations of Input and Controller drivers.
Also, if I try to open the Bluetooth menu inside Retroarh with the bluetoothctl driver, Retroarch close unexpectedly.

Thanks for working on this version and regards.

**Comment by basharast** [comment]
Hi, sorry for my late response,
In general Bluetooth functionality is not tested at all so probably it will not work
my test was on wireless controller with USB receiver.

as for Bluetooth menu it was mentioned before in the release note:
> Some features enabled for future development like "Bluetooth, Wifi..etc", they will crash by default

I will let you know once I start working on some solutions for that.

**Comment by namibiz** [comment]
Thanks a lot for your response. If you need some testing, please let me know.

Also, I did some testing connecting a Xbox Controller using an USB-C to USB-C cable, and Retroarch detects and works with it correctly, but button B must be the same as the **<-** (the Back button in the screen) and every time it's pressed, the Retroarch Quick menu comes up. I assume it's another different issue, but anyway I'll be glad to do any test you can need.

**Comment by basharast** [comment]
'B' event required for Windows Phone back press, otherwise the user will not be able to show the menu using touch screen.

I made this custom release so you can install it and it will not trigger menu on 'B' pressed
You can have both apps installed if you want to use the original one with touch.

https://mega.nz/file/X9JVhSqB#IIZBFbDnBrKFe7eSSGtv12peYIupaI1bt5akpYnrq8E

it will appear under title "RetroArch (Controller)"

**Comment by basharast** [comment]
True, this just a temporary solution, when I got more free time I will solve it with more accurate way.

**Comment by basharast** [comment]
Apparently bluetooth is not going to be an easy job to get it work, but as for `B` button this issue has been solved in the latest release [RetroArch 1.14.4](https://github.com/basharast/RetroArch-ARM/releases/tag/1.14.4) I add
